nevergrad/benchmark/experiments.py

```diff
@@ -892,6 +893,20 @@ def images(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
                     if not xp.is_incoherent:
                         yield xp
 
+@registry.register
+def adversarial_images(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Adversarial attacks."""
+    seedg = create_seed_generator(seed)
+    algos = ["CMA", "Shiwa", "DE", "PSO", "RecES", "RecMixES", "RecMutDE", "ParametrizationDE"]
+    for budget in [100 * 5 ** k for k in range(3)]:
+        for num_workers in [1]:
+            for algo in algos:
+                # Simple example with just the default case.
+                for func in [ImageAdversarial()]:
```

Comment:
this would need to be `ImageAdversarial.from_testbed(name)`. You won't be able to use it this way reliably.
Also, please instantiate the function before all the for loops, that will save a lot of computation.